l10n/en/products/vpn/platforms/android_new.ftl

```diff
@@ -0,0 +1,22 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/mobile/android
+
+vpn-android-vpn-for-android = VPN for { -brand-name-android } from { -brand-name-mozilla }
+vpn-android-ease-security = Ease Security Fears with { -brand-name-mozilla-vpn } for { -brand-name-android }
+vpn-android-looking-for-a = Looking for a VPN to protect your { -brand-name-android } device? { -brand-name-mozilla-vpn } is backed by a non-for-profit company. Learn about how a VPN protects you and why you should trust { -brand-name-mozilla } with your privacy and security.
+vpn-android-whether-youre = Whether you’re at home, at work, or making up a funny name for your coffee order at the register, protecting your online security is a top priority for { -brand-name-mozilla }. Like many smart technology products, { -brand-name-android } devices have built-in protection feaures that safeguards your device. With VPN for { -brand-name-android } from { -brand-name-mozilla }, you can take security further with just a tap of your finger.
+vpn-android-using-the-fast = Using the fast and secure { -brand-name-wireguard } VPN protocol, our VPN is able to encrypt the internet traffic between your device and the VPN network of servers. This helps us keep your web traffic protected when you are on the go. It’s like the lid protecting your green tea with honey from the fly buzzing around the table.
+vpn-android-how-does-a = How does a VPN protect my { -brand-name-android } device?
+vpn-android-vpns-help = VPNs help prevent:
+vpn-android-your-internet-activity = Your internet activity from being viewable, and then sold or shared with advertisers.
+vpn-android-low-video = Low video streaming quality due to slower and restricted bandwidth from your internet service provider (ISP)
+vpn-android-your-personal = Your personal data being stolen when using a public wi-fi network.
```

Comment:
Nit: capitalize "Wi-Fi" (not sure why, but it seems to be the established rule... there's apparently a styleguide somewhere.)